index.html

```diff
@@ -4297,19 +4304,60 @@ <h2>JSON Processing</h2>
 implementing the rule above. This can ensure proper term identification,
 typing, and order, when a JSON document is processed as JSON-LD.
           </p>
+          <p>
+However, the intention of the issuer is not preserved, unless the <code>@context</code>
+is applied, and the resulting <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>,
+and their exact RDF types, and node and edge structure is understood by the verifier.
+          </p>
 
           <p>
-The rule above guarantees semantic interoperability between JSON and JSON-LD for
-literal JSON keys mapped to URIs by the `@context` mechanism. While JSON-LD
-processors will use the specific mechanism provided and can verify that all
-terms are correctly specified, JSON-based processors implicitly accept the same
-semantics without performing any JSON-LD transformations, but instead by
+These rules alone do not guarantee semantic interoperability between JSON and JSON-LD for
+literal JSON keys mapped to URIs by the `@context` mechanism.
```

Comment:
I think this is true now with some clean up above and it's simpler:

```suggestion
The rules above guarantee semantic interoperability between applications
that are hard-coded to specific contexts and applications that more
generally consume JSON-LD documents.
```

Reply:
This is not true, but could be true modulo some defense language around contexts changing, and it would be nice to say it this way.

Reply:
@dlongley  @msporny I am not intending to accept this suggestion, in the interest of getting this PR to mergable state, I suggest we resolve any threads that we feel don't need further discussion on the PR, if you need issue markers to resolve a thread, please file issues, and I will address all issues filed in a subsequent PR.